control/tests/freqresp_test.py

```diff
@@ -423,7 +423,7 @@ def test_dcgain_consistency():
     sys_ss = ctrl.tf2ss(sys_tf)
     assert 0 in sys_ss.pole()
 
-    # Finite (real) numerator over 0 denominator => inf + nanj
+    # Finite (real) numerator over 0 denominator => inf
```

Comment:
The comment was correct and the checks that are done are against inf + nanj.  It is only `dcgain` that returns `inf`.